app/models/CrystallisationStatus.scala

```diff
@@ -33,18 +34,16 @@ case class CrystallisationStatus(status: String,
   }
 
   def createOverwriteCalculationListUrl: String = {
-    if (is1896) {
-      s"/income-tax/view/calculations/liability/$taxYearRange/$nino"
+    if (taxYear.is1896) {
```

Comment:
url manipulation is something related to controller / service or connector, but not case class, unless you planning to convert it to one of these